src/App.tsx

```diff
@@ -37,6 +39,16 @@ function App() {
                 throw new Error(message);
             }
             const data = await response.json();
+
+            data.records.sort((objectA:TodoItem, objectB:TodoItem) => {
+                const titleA = objectA.fields.title.toLowerCase();
+                const titleB = objectB.fields.title.toLowerCase();
+
+                if (titleA < titleB) return sortAsc ? -1 : 1;
+                if (titleA > titleB) return sortAsc ? 1 : -1;
+                return 0;
+            });
+
             const todos = data.records.map((todo: {id: number, fields: { title: string}}) => {
```

Comment:
After doing the above, we can make this `const todos: TodoItem[] = data.records.map((todo: TodoRecord) => { ...` instead to give a more clearly defined relationship to the response type!